docs/getting-started/usage/web-component.md

```diff
@@ -87,6 +87,15 @@ These properties can be provided as HTML attributes to the web component.
 />
 ```
 
+<!-- title: Hiding "Try It" -->
+
+```html
+<elements-api
+  apiDescriptionUrl="https://api.apis.guru/v2/specs/github.com/1.1.4/openapi.yaml"
+  hideTryIt
```

Comment:
I could not get this to work at all. Not sure if its a bug or if this is not how it works, but if you could look into it @mnaumanali94 I'd love to get an example up to make this easy for people.

Reply:
@philsturgeon Just to confirm: This works with `<elements-stoplight-project>` but not `<elements-api>`?